synapse/storage/database.py

```diff
@@ -435,7 +435,7 @@ def _do_execute(
             ):
                 return func(sql, *args, **kwargs)
         except Exception as e:
-            sql_logger.debug("[SQL FAIL] {%s} %s", self.name, e)
+            logger.error("[SQL FAIL] {%s} %s", self.name, e)
```

Comment:
Was this left over from debugging?

Reply:
I don't know why this is here...

```suggestion
            sql_logger.debug("[SQL FAIL] {%s} %s", self.name, e)
```